component/authenticator/pom.xml

```diff
@@ -0,0 +1,199 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><!--
+ ~ Copyright (c) 2016 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
As this is new connector make sure to Change the updated license header there. Check all other occurrences. As of now this should be WSO2 LLC. Check the license header from recently developed connectors. [1]

[1] - https://github.com/wso2-extensions/identity-oauth2-grant-rest/blob/main/pom.xml#L3-L18